<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWindowClass</class>
 <widget class="QMainWindow" name="AuthWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>210</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="authLabel">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Авторизация пользователя</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="passEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="passLabel">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Пароль</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="authButton">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="regButton">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Регистрация</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
